docs/tuning.md

```diff
@@ -122,21 +122,8 @@ large records.
 `R` is the storage space within `M` where cached blocks immune to being evicted by execution.
 
 The value of `spark.memory.fraction` should be set in order to fit this amount of heap space
-comfortably within the JVM's old or "tenured" generation. Otherwise, when much of this space is
-used for caching and execution, the tenured generation will be full, which causes the JVM to
-significantly increase time spent in garbage collection. See
-<a href="https://docs.oracle.com/javase/8/docs/technotes/guides/vm/gctuning/sizing.html">Java GC sizing documentation</a>
-for more information.
```

Comment:
Yeah let me actually use that link in favor of the existing one below. This one is the more recent one.